legal-api/tests/unit/services/test_authorization.py

```diff
@@ -1541,6 +1561,159 @@ def mock_auth(one, two):  # pylint: disable=unused-argument; mocks of library me
             assert filing_types == expected
 
 
+@pytest.mark.parametrize(
+    'test_name,state,legal_types,username,roles,completed_filing_types,completed_filing_sub_types,expected',
+    [
+        # active business - staff user
+        ('staff_active_cp_unaffected', Business.State.ACTIVE, ['CP'], 'staff', [STAFF_ROLE],
+         ['consentContinuationOut'], [None],
+         expected_lookup([FilingKey.ADMN_FRZE,
+                          FilingKey.AR_CP,
+                          FilingKey.COA_CP,
+                          FilingKey.COD_CP,
+                          FilingKey.CORRCTN,
+                          FilingKey.COURT_ORDER,
+                          FilingKey.VOL_DISS,
+                          FilingKey.ADM_DISS,
+                          FilingKey.REGISTRARS_NOTATION,
+                          FilingKey.REGISTRARS_ORDER,
+                          FilingKey.SPECIAL_RESOLUTION])),
```

Comment:
At this point, I think we only need to do corps(BC, BEN, CC, ULC) tests for this test.  Consent continuation out filings won't be allowed for non-corps and I think we have this covered in other tests.  So this scenario is not possible.  

Can we remove all tests that don't involve corps?  In the future, we may have other filings and entity types using `completedFilings` but for now its just the `continuationOut` filing using it.

Reply:
Yes, I will